.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: edvinas108/meme-grid:latest
+    steps:
```

Comment:
Should also include the steps which handle Gradle caching. Otherwise the build (tests) each time will have to re-download dependencies and such, which takes quite a lot of time.

More info (see the Gradle section):
https://circleci.com/docs/2.0/language-java

Reply:
included steps for gradle caching